configurations/test_telegram_group_pipeline_configuration.py

```diff
@@ -0,0 +1,113 @@
+from core_data_modules.cleaners import swahili
+from dateutil.parser import isoparse
+
+from src.pipeline_configuration_spec import *
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="engagement-db-telegram-group-test",
+    engagement_database=EngagementDatabaseClientConfiguration(
+        credentials_file_url="gs://avf-credentials/firebase-test.json",
+        database_path="engagement_db_experiments/experimental_test"
+    ),
+    uuid_table=UUIDTableClientConfiguration(
+        credentials_file_url="gs://avf-credentials/firebase-test.json",
+        table_name="_engagement_db_test",
+        uuid_prefix="avf-participant-uuid-"
+    ),
+    operations_dashboard=OperationsDashboardConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-dashboards-firebase-adminsdk-gvecb-ef772e79b6.json",
+    ),
+    telegram_group_sources=[
+        TelegramGroupSource(
+            token_file_url="gs://avf-credentials/dev-telegram-tokens.json",
```

Comment:
```suggestion
            token_file_url="gs://avf-credentials/dev-telegram-token.json",
```
?

Reply:
This contains more than one credential (api_id,  api_hash, phone_number) so best to keep it as tokens